Foundation/NSTimeZone.swift

```diff
@@ -212,12 +211,17 @@ extension NSTimeZone {
             return dictionary._nsObject._bridgeToSwift() as! [String : String]
         }
         set {
-            // CFTimeZoneSetAbbreviationDictionary(newValue._cfObject)
-            NSUnimplemented()
+            CFTimeZoneSetAbbreviationDictionary(newValue._cfObject)
         }
     }
 
-    open class var timeZoneDataVersion: String { NSUnimplemented() }
+    open class var timeZoneDataVersion: String {
+        #if os(Windows)
+            return "" // We do not source timezone data from ICU. The empty string is what Darwin would return if ICU isn't able to give a time zone data version.
```

Comment:
Hmm, we should be able to do this on Windows I think.  We do have ICU available and will link against it.

Reply:
Ah, okay. Do you want me to just return the ICU version?

Reply:
Yes please!